store/account.go

```diff
@@ -119,10 +119,7 @@ func (s *AutonomyStore) UpdateAccountGeoPosition(accountNumber string, latitude,
 	}
 
 	if exist {
-		err = s.mongo.UpdateAccountGeoPosition(a.AccountNumber, latitude, longitude)
-		if nil != err {
-			return err
-		}
+		return s.mongo.UpdateAccountGeoPosition(a.AccountNumber, latitude, longitude)
```

Comment:
This fix is not related to this PR.

Previously if `err == nil`, the function doesn't exit and falls through to the function which attempts to create a new account.